.agent/workflows/develop.md

```
@@ -0,0 +1,212 @@
+---
+description: Implement requirements from a clarified document using repository skills and AGENTS.md guidance
+---
+
+## Table of Contents
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)